### DISCUSSION TITLE
How do you measure the quality of translation?

### DISCUSSION THREAD

**Discussion by codehooni** [goatshriek/stumpless#414] [General]
I would like to localize Korean in the stubless project. I have a question during the process. How do you judge the quality of the translation? Is there a team that verifies it separately?

**Comment by goatshriek** [comment]
That's a great question! Because we don't have language skills in all of the languages that have been contributed, we rely on the community to review and correct existing translations. #156 and the related #168 is an example of one such review.

We do ask that you have some language knowledge and not simply use a machine translator, since the results of this could be by unhelpful or misleading as they would occur out of context.
